Glossary.md

```diff
@@ -0,0 +1,83 @@
+# Glossary
+
+The glossary provides definitions for our team to have a shared language around
+FOIA. This is mostly for use within our team.
+
+
+##### Chief FOIA Officer
+
+The person with agency-wide responsibility for oversight and support to FOIA
+programs. The position is required by the Freedom of Information Act and
+includes several additional responsibilities.
+
+
+##### cms
+
+See [case management-system](#case-management-system).
+
+
+##### case management system
+
+A system (usually electronic) to track the workflow of individual FOIA requests
+(cases). Each agency can have their own case management system they use to
+triage, assign, track, and fulfill requests. Agencies may use case management
+systems for as much or as little of the FOIA fulfillment process as they wish.
+
+
+##### disclosability
+
+The degree to which a piece of information is disclosable. Often used when
+evaluating what information in a record is disclosable or what must be redacted.
+
+
+##### FOIA
+
+The Freedom of Information Act which provides that any person has a right,
+enforceable in court, to obtain access to federal agency records, except to the
+extent that any portions of such records are protected from public disclosure by
+one of nine exemptions or by one of three special law enforcement record
+exclusions.
+
+
+##### FOIA Public Liaison
+
+Designated by the Chief FOIA Officer who is responsible for acting as the public
+contact for addressing concerns of requesters related to FOIA administration.
+
+
+##### PII
+
+Personally Identifiable Information.
+
+> Information that can be used to distinguish or trace an individual’s identity,
+> either alone or when combined with other personal or identifying information
+> that is linked or linkable to a specific individual.
+
+_-- [GSA Privacy Program](https://www.gsa.gov/portal/content/104256)_
+
+The definition of PII is often context specific and requires a case-by-case
+assessment. Sometimes when combined with other information, non-PII can become
+PII.
+
+
+##### portal
+
+A gateway to other systems or information but may not have any functionality or
+information on its own.
+
+
+##### proactive disclosure
+
+
+##### release to three, release to all
```

Comment:
Per the FOIA 1996 amendment, if a set of records is requested three times, the FOIA indicates the documents should be made permanently public, which forms the basis of the [“reading room” provision](https://www.justice.gov/oip/blog/foia-post-2003-foia-counselor-qa-frequently-requested-records).